backend/src/main/java/de/wolffclan/backend/models/date/NewDate.java

```diff
@@ -0,0 +1,24 @@
+package de.wolffclan.backend.models.date;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+
+public record NewDate(
+        LocalDate meetingDay,
+        LocalTime meetingTime,
+        String description,
+        int maxUser,
+        int userCounter,
+        List<String> userIds,
+        String placeId
+) {
+    public boolean isValid() {
+        return (this.meetingDay != null)
+                && (this.meetingTime != null)
+                && (this.maxUser >= 1)
+                && (this.userCounter >= 1)
+                && (this.userIds != null)
+                && (this.placeId != null);
```

Comment:
Auf die `this`s kannst Du auch getrost verzichten. 😄

Reply:
Ich glaube das war wegen Sonar. Gaaanz tooooles Program.